src/pages/login/index.jsx

```diff
@@ -1,22 +1,61 @@
-import { useState } from 'react';
+import { useState, useContext, useEffect } from 'react';
+import Cookies from 'universal-cookie';
+import { AuthContext } from '../../contexts/AuthContext';
 import { Button, Input } from '@mui/material';
 import styles from './index.module.css';
 import ApplicationLogin from '../../common/layouts/ApplicationLogin';
 import Link from '../../common/components/Link';
+import HelpText from '../../common/components/HelpText';
 
 const buttonSX = {
   margin: '20px 0px 10px 0px',
 };
 
 function Login() {
+  const [user, setUser] = useContext(AuthContext);
   const [emailValue, setEmailValue] = useState('');
   const [passwordValue, setPasswordValue] = useState('');
+  const [error, setError] = useState(false);
+  const [textError, setTextError] = useState('');
+  const [disabledButton, setDisabledButton] = useState(true);
+
+  useEffect(() => {
+    if (emailValue && passwordValue) {
+      setDisabledButton(false);
+    } else setDisabledButton(true);
+  }, [emailValue, passwordValue]);
```

Comment:
We can achieve the same result by passing `disabled={isLoading || !(emailValue && passwordValue)}` to the Button without using an extra useEffect.